.github/workflows/ui_tests.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: UI tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  source-build:
+    name: Run Netdata Agent UI tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git clone repository
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Install the agent
+        run: |
+          ./packaging/installer/install-required-packages.sh netdata --non-interactive --dont-wait
+          sudo ./netdata-installer.sh --dont-wait
```

Comment:
Should we add `--disable-telemetry`? Otherwise, if I understand it right, we will send statistics to the PostHog on every job run.

Reply:
Excellent catch.

Reply:
See the changes in https://github.com/netdata/netdata/pull/11233 for an example of what to change here to just disable telemetry for the whole workflow.